library/psa_crypto.c

```diff
@@ -4463,6 +4363,15 @@ psa_status_t psa_key_derivation_abort( psa_key_derivation_operation_t *operation
             mbedtls_free( operation->ctx.tls12_prf.label );
         }
 
+#if defined(MBEDTLS_PSA_BUILTIN_ALG_TLS12_PSK_TO_MS)
```

Comment:
Please add a reference to the upstream PR from the fromlist commit.

For tracking and review purposes.